File: codes/java/fibonacci/Fibonacci.java
import java.util.Scanner;

class Fibonacci {
    public static void main(String[] args) {
        int first=0, second=1, n, i, next;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter no. of terms: ");
        n = sc.nextInt();
        sc.close();
        System.out.println("Fibonacci series are: ");
        for(i=0; i<n; i++) {
            System.out.print(first + " ");
            next = first + second;
            first = second;
            second = next;
        }
    }
}

File: codes/c++/fibonacci/fibonacci.cpp
#include<iostream>
using namespace std;

int main() {
    int n, i=0, first=0, second=1, next;
    cout<<"Enter number of terms: ";
    cin>>n;
    do {
        cout<<first<<" ";
        next = first + second;
        first = second;
        second = next;
        i++;
    } while(i<n);
    return 0;
}